js_modules/dagster-ui/packages/ui-core/src/asset-selection/input/util.ts

```diff
@@ -17,13 +20,7 @@ export const getAttributesMap = (assets: AssetGraphQueryItem[]) => {
       if (isKindTag(tag)) {
         return;
       }
-      if (tag.key && tag.value) {
-        // We add quotes around the equal sign here because the auto-complete suggestion already wraps the entire value in quotes.
-        // So wer end up with tag:"key"="value" as the final suggestion
-        tagNamesSet.add(`${tag.key}"="${tag.value}`);
-      } else {
-        tagNamesSet.add(tag.key);
-      }
+      tagNamesSet.add(memoizedTag(tag.key, tag.value));
```

Comment:
Stupid nit but this seems like a `tagPairsSet` or `tagsSet` , since it includes the values.